Makefile

```diff
@@ -63,13 +64,29 @@ react-app-test: | $(REACT_APP_NODE_MODULES_PATH) react-app-lint
 	@echo ">> running React app tests"
 	cd $(REACT_APP_PATH) && yarn test --no-watch --coverage
 
+.PHONY: module-build
```

Comment:
In the overall context of Prometheus, the term `module` is a bit too unspecific IMO. How about `npm-module` or `web-module` as a prefix for these?

Reply:
yeah you are right I will change the name.